### PR TITLE
Add week 03 meeting

### PR DESCRIPTION
**Pull request by Tjzabel**
This PR is for the upcoming week 3 meeting on networking.

### CODE REVIEW

File: announcements/_posts/2020-09-04-w03-networking.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+title: "Week #3: Networking"
+layout: post-event
+date-start: "2020-09-04 16:30"
+date-end: "2020-09-04 18:00"
+location: "Discord"
+---
+
+Hey RITlug!
+
+For this week's meeting, we will have a recent RIT graduate, Nathaniel Larrimore, who will be giving a presention on Linux networking!
```
Comment (jrtechs) [inline announcements/_posts/2020-09-04-w03-networking.md:12]: "presentation". It is not me this time :)

**Review by jrtechs — CHANGES_REQUESTED** [review]
-->""

File: announcements/_posts/2020-09-04-w03-networking.md (hunk 1 above)
Comment (Tjzabel) [reply announcements/_posts/2020-09-04-w03-networking.md:12]: presention sounds better

**Review by jrtechs — APPROVED** [review]
LGTM

**Comment by Tjzabel** [comment]
Email sent.

**Pull request merged**